src/templates/article.js

```diff
@@ -177,10 +178,16 @@ const Article = props => {
         <Layout>
             <Helmet>
                 <title>{articleTitle}</title>
-                <meta property="og:title" content={articleTitle} />
-                <meta property="og:url" content={articleUrl} />
-                <meta property="og:type" content="article" />
-                <link rel="canonical" href={articleUrl} />
+                <meta
+                    property="og:image"
+                    content={
+                        og.image || `${siteUrl}${withPrefix(meta['atf-image'])}`
+                    }
+                />
+                <meta property="og:title" content={og.title || articleTitle} />
```

Comment:
Are the attributes definitely coming back in `meta`? The twitter directive gets passed through the factory.

Reply:
They are, Sophie asked me where we would like to put them and I thought meta was best based on how we are using them now.

<img width="1253" alt="Screen Shot 2020-03-27 at 4 16 12 PM" src="https://user-images.githubusercontent.com/9064401/77797219-16755a80-7047-11ea-8eb2-c8cdcf6f5453.png">